xbmc/platform/win10/PlatformWin10.cpp

```diff
@@ -45,3 +45,12 @@ void CPlatformWin10::PlatformSyslog()
 {
   CWIN32Util::PlatformSyslog();
 }
+
+bool CPlatformWin10::SupportsUserInstalledBinaryAddons()
+{
+#if defined(TARGET_WINDOWS_STORE)
```

Comment:
PlatformWin10.cpp is only executed/compiled on UWP then is not needed `#if defined(TARGET_WINDOWS_STORE)`

can return always `false` here and Windows desktop uses default code path in `xbmc/platform/Platform.h`

Reply:
Thanks. Changed.